### PR TITLE
POL-692 Azure Instance Rightsizing - Merge Idle Instances and Inefficient Instance Utilization

### PR DESCRIPTION
**Pull request by nia-vf1**
initial commit

### Description

[Describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Review by taterdog590 — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ Reference
 - [Azure Expiring Reserved Instances](./cost/azure/reserved_instances/expiration)
 - [Azure Reserved Instance Utilization](./cost/azure/reserved_instances/utilization/)
 - [Azure Reserved Instance Recommendations](./cost/azure/reserved_instances/mca_recommendations)
+- [Azure Rightsize Compute Instances](./cost/azure/rightsize_compute_instances)
 - [Azure Unused IP Addresses](./cost/azure/unused_ip_addresses)
 - [Azure Savings Realized from Reservations](./cost/azure/savings_realized)
 - [Azure Schedule Instance](./cost/azure/schedule_instance/)
```
Comment (bryankaraffa) [inline README.md:110]: Should these have been removed ?

File: cost/azure/rightsize_compute_instances/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## v3.0
```
Comment (bryankaraffa) [inline cost/azure/rightsize_compute_instances/CHANGELOG.md:3]: V1.0 - explained better on another comment but happy to discuss further with the team (Michael, Lawrence, Shawn, etc..)

File: README.md (hunk 1 above)
Comment (nia-vf1) [reply README.md:110]: Which are you referring to?

File: cost/azure/rightsize_compute_instances/CHANGELOG.md (hunk 2 above)
Comment (nia-vf1) [reply cost/azure/rightsize_compute_instances/CHANGELOG.md]: Updated to v1.0

**Review by taterdog590 — APPROVED** [review]
LGTM

**Pull request merged**